src/v/cluster/commands.h

```diff
@@ -330,6 +335,21 @@ using feature_update_license_update_cmd = controller_command<
   model::record_batch_type::feature_update,
   serde_opts::serde_only>;
 
+// Plugins
+using transform_update_cmd = controller_command<
+  int8_t, // unused
+  model::transform_metadata,
+  transform_update_cmd_type,
+  model::record_batch_type::plugin_update,
+  serde_opts::serde_only>;
+
+using transform_remove_cmd = controller_command<
+  model::transform_name,
+  int8_t, // unused
```

Comment:
the int8 is in different places for the 2 commdns, update+remove

Reply:
for remove, the name is the key and we don't need a value, but for update the name is in the metadata value so putting it in the key field would be redundant.

Reply:
+1